modules/tinymce/CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `GetContent` event was not fired when getting `tree` or `text` formats using the `editor.selection.getContent()` API #TINY-8018
 - The `table` plugin would sometimes not correctly handle headers in the `tfoot` section #TINY-8104
 - Deleting content would sometimes not fire `input` events as expected #TINY-8168
+- Deleting content would sometimes not fire `BeforeInput` events as expected #TINY-8329
```

Comment:
```suggestion
- Deleting content would sometimes not fire `beforeinput` and `input` events as expected #TINY-8168 #TINY-8329
```
This can be collapsed to a single line